docs/src/main/sphinx/installation/cli.rst

```diff
@@ -381,10 +383,39 @@ press :kbd:`Enter`.
 By default, you can locate the Trino history file in ``~/.trino_history``.
 Use the ``TRINO_HISTORY_FILE`` environment variable to change the default.
 
+Batch mode
+----------
+
+Running the Trino CLI with the ``--execute``, ``--file`` or passing queries to
+the standard input uses the batch mode (non-interactive), in which
+the Trino CLI exits after executing all queries, does not report progress,
+and prints results in the ``CSV`` format.
+
+The following options are available to further configure the CLI in batch
+mode:
+
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
+
+  * - Option
+    - Description
+  * - ``--execute=<execute>``
+    - Execute specified statements and exit.
+  * - ``-f``, ``--file=<file>``
+    - Execute statements from file and exit.
+  * - ``--ignore-errors``
+    - Continue processing in batch mode when an error occurs (default is to
+      exit immediately).
+  * - ``--output-format=<format>``
+    - :ref:`cli-output-format` for batch mode.
```

Comment:
Specify the format to use for printing query results. 

"for batch mode" is implied and can be left out

Reply:
Done!